crates/driver/src/infra/config/mod.rs

```diff
@@ -14,7 +14,7 @@ pub struct Config {
     pub disable_access_list_simulation: bool,
     pub disable_gas_simulation: Option<eth::Gas>,
     pub solvers: Vec<solver::Config>,
-    pub liquidity: liquidity::Config,
+    pub liquidity: LiquidityConfig,
```

Comment:
This isn't great. This `Config` type isn't supposed to know anything about the config files. It's just a type which collects all infra config objects. So this should be `liquidity::Config` like it was before...

Reply:
We did this with `ContractAddresses`, so I took the same approach here.

Essentially, it needs to be slightly different because runtime values are needed to do the mapping correctly. Specifically (like for `ContractAddresses`, the `eth::NetworkId` needs to be known.

Alternatively, and I played with this as well, but also didn't feel super clean, was to pass in the `eth::Ethereum` or `eth::NetworkId` to the `config::load` function. This would allow both `ContractAddresses` and `LiquidityConfig` to not leak.

Reply:
Hmm, OK. I agree that passing the `eth` object in `load` is not the best. I think maybe it's time to jettison this `Config` type entirely. I don't expect you to do that though (at least not in this PR).